history/tools.py

```diff
@@ -44,36 +47,39 @@ def normalization(data, new_max=1, new_min=0):
             data[i] = (((data[i] - old_min) * new_range) / old_range) + new_min
     return data
 
-def filter_by_mins(data,mins=5):
+
+def filter_by_mins(data, mins=5):
     new_data = []
 
     for ele in data:
         ele_unixtime = int(time.mktime(ele.created_on.timetuple()))
-        ele_unixtime_min = int(ele_unixtime/60)
+        ele_unixtime_min = int(ele_unixtime / 60)
         if ele_unixtime_min % mins == 0:
             new_data.append(ele)
 
     return new_data
 
+
 def median_value(queryset, term):
     count = queryset.count()
-    return queryset.values_list(term, flat=True).order_by(term)[int(round(count/2))]
+    return queryset.values_list(term, flat=True).order_by(term)[int(round(count / 2))]
+
 
-def get_exchange_rate_to_btc(ticker,d=None):
+def get_exchange_rate_to_btc(ticker, d=None):
     from history.models import Price
-    
+
     if d is None:
         d = datetime.datetime.now()
     if ticker == 'BTC':
         return 1
-    
+
     try:
-        exchange_pair = 'BTC_'+ticker
-        latest_price = Price.objects.filter(symbol=exchange_pair,created_on__lt=d).order_by('-created_on').first()
+        exchange_pair = 'BTC_' + ticker
+        latest_price = Price.objects.filter(symbol=exchange_pair, created_on__lt=d).order_by('-created_on').first()
         return latest_price.price
     except:
```

Comment:
only if there aren't any prices loaded yet, which is fair case for the entire system.  or if there's an invalid exchagne_pair